### DISCUSSION TITLE
Code formatting style in C files

### DISCUSSION THREAD

**Discussion by isturiz** [neovim/neovim#31586] [Q&A] [answered]
Out of curiosity, I was reading the source code and noticed that there are several switch statements with inconsistent indentation, even within the same file like src/vterm/parser.c, where different indentation styles coexist.

Is there an intentional reason behind this?

I’m not an expert in C, but it struck me as a bit odd. I also tried running `make formatc` and `make format` hoping that it might apply some changes, but nothing happened.

Before creating an Issue or a PR (as this seems like a minor detail), I thought I’d ask here first.

Thank you in advance for any responses!

**Comment by clason** [comment, marked as answer]
`vterm` is a vendored external dependency; it's not (yet, fully) subject to the code style enforced by our formatter. (Intentionally, I should add, to make backporting possible upstream changes easier.)

**Reply by isturiz** [reply]
Thank you for the clarification! That makes sense.
